static/js/apps/eval_retrieval_generation/constants.tsx

```diff
@@ -68,3 +70,7 @@ export const QUERY_OVERALL_OPTION_IRRELEVANT = "LLM_ANSWER_IRRELEVANT";
 export const QUERY_OVERALL_OPTION_SOMEWHAT_RELEVANT =
   "LLM_ANSWER_SOMEWHAT_RELEVANT";
 export const QUERY_OVERALL_OPTION_RELEVANT = "LLM_ANSWER_RELEVANT";
+// Options for the query calls overall feedback
+export const OVERALL_QUESTIONS_OPTION_HAS_MISSING = "LLM_QUESTIONS_MISSING";
```

Comment:
the constants starts to feel overwhelming, later on may think about simplifying them. Imagine if half of these need change, what would the change look like (or minimized).

Reply:
added a TODO to clean up the constants